packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -63,8 +64,6 @@ contract Bridge is EssentialContract, IBridge {
         _;
     }
 
-    receive() external payable { }
```

Comment:
if ether_vault is non-zero, the the Bridge shall not be allowed to receive token directly.

Reply:
This causes the tests to fail. Why ? The EtherVault needs to send the funds from EtherVault to the Bridge. This code is needed.

Reply:
Will add back in my test fix branch.